roles/provision-metrics-apb/files/mobile-services-dashboard.json

```diff
@@ -1171,6 +1171,516 @@
       ],
       "title": "Mobile app metrics",
       "type": "row"
+    },
+    {
+      "collapsed": false,
```

Comment:
Others are `true`. Makes the security group be the only one open by default right now.

Reply:
+1